### ISSUE TITLE
Script Safety Suggestion

### ISSUE THREAD

**Issue by SomethingGeneric** [emmett1/lfs-scripts#21]
Hey! Awesome script.

I'm still in the process of building a system with this on Arch, but I'd love it if you were to change the location of the tools folder, to say `/home/$USER/tools` or something. At the very least, make it a configurable option in the `config` file. 

It's not so much about conforming to LSB or POSIX, and more about not adding random folders in the root directory without telling the user. (Or did I miss that somewhere?) Just a thought. You might get a PR for this soon. :P

**Comment by emmett1** [comment]
Yeah i'm planning to change tools directory into working directory but /tools symlink still required.

So far this script does not randomly create files or directory other than /tools and /mnt/lfs. I'm thankful if you willing to help anything by making PR since i dont have much time lately to fully maintaining this scripts.
